module/htdocs/js/shinken-actions.js

```diff
@@ -599,9 +599,11 @@ $("body").on("click", ".js-recheck", function () {
     var elt = get_action_element($(this));
 
     if (elt) {
+        console.log(elt);
         recheck_now(elt);
     } else {
         $.each(selected_elements, function(idx, name){
+            console.log("Recheck selected", elt);
```

Comment:
I don't see how this is relevant with the rest of the PR.

Reply:
Oops :wink: I will revert this